connectivity/cycles.py

```diff
@@ -2,7 +2,7 @@
 import networkx as nx
 
 zipCode = '02138'

```

Comment:
Correct. Neither of the .py scripts are in use currently. Was a vestige of initial development as the tool was starting to take shape and before we settled on pg_routing.